docs/content/docs/user-docs/leader-election.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Leader election"
```

Comment:
Hesitant to name this `Managing Controller Leadership`

Reply:
All of our other docs use a verb, so I would prefer it to match that format. My suggestion is: `Managing multiple instances of ACK with leader election`?